updater/src/Node/FS/Aff/Extra.purs

```diff
@@ -0,0 +1,56 @@
+module Node.FS.Aff.Extra
```

Comment:
This file has been added because I need to use a behavior for `mkdir` supported by Node but not supported by the way the `mkdir` function was implemented in the Node bindings.